src/main/java/com/danielagapov/spawn/Models/User.java

```diff
@@ -18,12 +20,13 @@ public class User implements Serializable {
         @Id
         @GeneratedValue
         private UUID id;
-
+        private List<User> friends;
```

Comment:
Leave this out of the entity. It should only be in the DTO. Refer to the README's DTO drawing, as this case happens to exactly match it. 

Basically, `friends` as a list can't actually be put directly into SQL, which is what this `@Entity` tag does behind the hood with Spring Data JPA to a MySQL table. 

So, while we need the `friends` property in the DTO, we need to fetch that appropriately in the mapper files, not from the entity directly.